lnwallet/interface_test.go

```diff
@@ -1227,6 +1229,26 @@ func testListTransactionDetails(miner *rpctest.Harness,
 			t.Fatalf("num confs incorrect, got %v expected %v",
 				txDetail.NumConfirmations, 0)
 		}
+
+		// We test that each txDetail has destination addresses. This ensures
+		// that even when we have 0 confirmation transactions, the destination
+		// addresses are returned.
+		if len(txDetail.DestAddresses) == 0 {
```

Comment:
this can be removed now, covered by below check

Reply:
fixed